pkg/streamingpromql/testdata/ours/aggregators.test

```diff
@@ -327,3 +327,18 @@ eval range from 0m to 1m step 1m count(series)
 
 eval range from 0m to 1m step 1m group(series)
   {} _ 1
+
+clear
+
+load 1m
```

Comment:
Should we make this 6m instead to avoid lookbacks? 

The way the test data is constructed below, it looks like the goal is to test scenarios where there is only one input sample for a group (eg. the 5th step for group A).

Or is the intention to exercise lookbacks?

Reply:
The intention was to exercise, but it could be worth doing for only one input series. I'll add another group with single points for that.

Reply:
I added this check upstream that we'll eventually get. If I were to add this here I would have to comment it out until upstream is merged anyway.